modules/tutorials/pages/migration-tutorial/sql-migration-tutorial-couchbase-server.adoc

```diff
@@ -0,0 +1,150 @@
+= Migrating your Data from MySQL to Couchbase Server
+:description: Using MySQL as a starting point, this guide demonstrates \
+how to migrate your existing data from SQL tables to documents stored in a Couchbase bucket.
+:page-topic-type: guide
+:page-pagination: next
+
+[abstract]
+{description}
+
+== Introduction
+
+Couchbase offers a number of strategies for migrating your existing data; 
+in this example, we will begin with a sample student record database stored in MySQL, 
+and use the Couchbase server tool `cbimport` to copy the data into a Couchbase cluster.
+
+== Prerequisites
+
+Before you begin this exercise, you should have installed and set up a Couchbase cluster on your local machine.
+You will find instructions for creating a fresh cluster here: xref:getting-started:do-a-quick-install.adoc[Couchbase Server Installation]
+
+To use `cbimport`, you will need to install the Couchbase `CLI` package. 
+You will find the location of the package and instructions for installing it, here: xref:cli:cli-intro.adoc[]
+
+If you're running through the examples, 
+then you will also need an existing MySQL installation with the preexisting table structure 
+defined in xref:student-record-sql-database-section[the following section].
+
+IMPORTANT: This tutorial makes use of the MySQL JSON functions that were introduced in version `5.7.22`.
+Make sure you have installed MySQL version `5.7.22` or later.
+
+include::partial$/migration-tutorial/student-database-example.adoc[]
+
+include::partial$/migration-tutorial/extract-sql-data.adoc[]
+
+
+== Create your bucket, scope, and collections.
+
+You will need to create the bucket, scope, and collections to hold the data on your Couchbase cluster.
+
+For information on creating buckets, scopes, and collections,
+read the sections on xref:manage:manage-buckets/bucket-management-overview.adoc[Managing Buckets]
+and xref:manage:manage-scopes-and-collections/manage-scopes-and-collections.adoc[Managing Scopes and Collections]
+
+'''
```

Comment:
I didn't know we used this `<hr />` type thing.
Interesting to see.

Reply:
I think I've seen Sarah use it in a few places.